src/main.ts

```diff
@@ -44,7 +44,9 @@ async function main() {
 	const { TYPES_BUNDLE, TYPES_SPEC, TYPES_CHAIN, TYPES } = config.SUBSTRATE;
 	// Instantiate a web socket connection to the node and load types
 	const api = await ApiPromise.create({
-		provider: new WsProvider(config.SUBSTRATE.WS_URL),
+		provider: config.SUBSTRATE.HTTP_URL
```

Comment:
Since we have to check and decide on which env var URL we're using here, how about passing the "for sure" url into `startUpPrompt` rather than having it also do the check?